CHANGELOG.md

```diff
@@ -105,6 +105,22 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Metroid Prime
 
+- Fixed: Crashes in several rooms when skipping cutscenes exactly 1 second from the end
+- Fixed: Incorrect shield texture for vertical **PowerBeamOnly** doors
+- Fixed: Elite Research - Stutter when loading room
+- Fixed: Ruined Shrine Access - Still playing Beetle fight music after leaving
+- Fixed: Research Entrance - Soflock if player kills pirates before touching the cutscene trigger (reverse labs)
+- Fixed: Mine Security Station - Wave Pirates not dropping down if the player didn't skip the cutscene immediately
+- Fixed: Uncrashed Frigate - Escape music sometimes playing at inappropriate times
+- Fixed: Furnace - Ghost elements re-appearing when re-entering from East Furnace Access
+- Fixed: Ruined Courtyard - Water platforms breaking when skipping cutscene on the last possible frame
+- Changed: Updated tournament winners scan
```

Comment:
feel free to ignore, but it looks more visually pleasing if all the changed ones come before the fixed.

Reply:
A really long time ago @henriquegemignani told me he prefers fixed first

Reply:
Preferred was something like:

```
Fixed
Added
Changed
Removed
```

Reply:
looked at what keepachangelog recommends, which is this:
```
added
changed
deprecated
removed
fixed
security
```

Which looks worse to me. but also who really cares about the order.

Reply:
let's both agree that anything is better than mixing/alternating types